skyportal/models/user_notification.py

```diff
@@ -21,9 +22,28 @@
 from .comment import Comment
 from .listing import Listing
 from .facility_transaction import FacilityTransaction
+from ..email_utils import send_email
+from twilio.rest import Client as TwilioClient
+import gcn
+from sqlalchemy import or_
+
+from skyportal.models import Shift, ShiftUser
 
 _, cfg = load_env()
 
+log = make_log('notifications')
+
+account_sid = cfg["twilio.sms_account_sid"]
```

Comment:
Will this still work for whatever is happening here? https://github.com/skyportal/skyportal/pull/3091

Reply:
What do you mean by that ? I mean it should work, the key exists, but it has no value by default. What should we do, add a default null or ~ value in the config ? My PR aside, that's a problem in general with empty fields in the config.

Reply:
Yeah I think it's fine.